pkg/sql/codegen/tensorflow/codegen.go

```diff
@@ -312,8 +312,11 @@ func Pred(predStmt *ir.PredictStmt, session *pb.Session) (string, error) {
 			fmt.Sprintf("\"%s\": [%s]", target, strings.Join(perTargetFeatureColumnsCode, ",\n")))
 	}
 	isKeras, estimatorStr := IsKerasModel(predStmt.TrainStmt.Estimator)
-	labelFM := predStmt.TrainStmt.Label.GetFieldMeta()[0]
-	if labelFM.Name == "" {
+	var labelFM *ir.FieldMeta
+	if predStmt.TrainStmt.Label != nil {
+		labelFM = predStmt.TrainStmt.Label.GetFieldMeta()[0]
+		labelFM.Name = predStmt.ResultColumn
```

Comment:
Seems this PR only keeps one label column that specified in the predict SQL?

Reply:
Yes, always using `predict result column` as the prediction result column name.